libs/core/src/lib/utils/functions/key-util.ts

```diff
@@ -97,6 +97,10 @@ export class KeyUtil {
                     return event.code
                         ? event.code.toLowerCase().includes('key')
                         : event.keyCode >= 65 && event.keyCode <= 90;
+                case 'control':
```

Comment:
what does `control` type mean in this case?

Reply:
Maybe this bad naming... I mean some service commands as  alt, ctr, insert, print, f4 etc. All except letters, numbers and symbols.